docs/upgrading.rst

```diff
@@ -883,6 +883,17 @@ In marshmallow 2, it was possible to have multiple fields with the same ``attrib
     MySchema()
     # No error
 
+
+`python-dateutil` recommended dependency is removed
+***************************************************
+
+In marshmallow 2, `python-dateutil` was used to deserialize RFC or ISO 8601
+datetimes if it was installed. In marshmallow 3, datetime deserialization is
+done with no additional dependency.
+
+`python-dateutil` is not a recommended dependency anymore.
```

Comment:
```suggestion
``python-dateutil`` is no longer used by marshmallow.
```

Reply:
I applied this change.

I never know when to use '`' and when to use '``'. I don't think we're through the docs.

Reply:
Double-backticks is for simple, pre-formatted text. Single-backticks is used when linking to other objects (e.g. autodocc'd or intersphinx members).

Reply:
Then there are a lot of single-backticks uses with no link in the docs (at least in upgrading.rst) where double-backticks should be used instead.

We can fix this when we get the time.